### PR TITLE
Feature form

### PR DESCRIPTION
**Pull request by serdg0**

### CODE REVIEW

File: .idea/.generators [hunk 1]
```diff
@@ -5,4 +5,4 @@ You are allowed to:
 2. Remove generators
```
Comment (mcrd25) [inline .idea/.generators:5]: The .idea folder should be added to your .gitignore file. Please add it before the next review. 🙂

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :name, presence: true
+  validates :email, presence: true
```
Comment (mcrd25) [inline app/models/user.rb]: Maybe you can add additional validation for uniqueness. **[OPTIONAL]**
I can currently create multiple users with the same email and username. see image below
Screen Shot 2019-09-06 at 8 57 00 AM

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<% if false %>
```
Comment (mcrd25) [inline app/views/users/new.html.erb:1]: This is a neat way of not showing these forms. Even though the instructions say to comment it out. I'd accept this ✔️ .

Very clever 😄

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new(name: '', email: '', password: '')
```
Comment (mcrd25) [inline app/controllers/users_controller.rb]: could simply have `@user = User.new`

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Great Job so far

Hi, @serdg0  and @MephistoDevelop 👋,

Okay team, let's get into this review by highlighting the awesome things you did
- Your README is very informative.  I commend you for the effort you put into it 👏. If you'd like to improve the visuals of your README checkout this [markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for useful syntax. 
- You were clever in some areas 😛 
- Did the basics well and stuck to the task at hand!


## Required Changes
- Handle your .gitignore issues. If you are having time removing already committed check out my gist on the topic and you should be fine. Just click [here](https://gist.github.com/mcrd25/1b2718054b65f4720fa582c6a4032217). **Files not related to the project should not be committed**

## Additional Suggestions [OPTIONAL]
- Maybe you can create a form partial and reuse it for `new` and `edit`

## Conclusion
*This project is well done! 👍 There is only one required issue for you to address above ⬆️. However, I do recommend that you Address some of the OPTIONAL suggestions*

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

**Review by mosaaleb — APPROVED** [review]
### Well done @serdg0, @MephistoDevelop :clap:

You can merge to master :+1: 

Good luck with the next projects :computer:

**Pull request merged**